plugins/modules/keycloak_authz_custom_policy.py

```diff
@@ -0,0 +1,211 @@
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_authz_custom_policy
+
+short_description: Allows administration of Keycloak client custom Javascript policies via Keycloak API
+
+version_added: 7.4.0
+
+description:
+    - This module allows the administration of Keycloak client custom Javascript via the Keycloak REST
+      API. Custom Javascript policies are only available if a client has Authorization enabled and if
+      they have been deployed to the Keycloak server as JAR files.
+
+    - This module requires access to the REST API via OpenID Connect; the user connecting and the realm
+      being used must have the requisite access rights. In a default Keycloak installation, admin-cli
+      and an admin user would work, as would a separate realm definition with the scope tailored
+      to your needs and a user having the expected roles.
+
+    - The names of module options are snake_cased versions of the camelCase options used by Keycloak.
+      The Authorization Services paths and payloads have not officially been documented by the Keycloak project.
+      U(https://www.puppeteers.net/blog/keycloak-authorization-services-rest-api-paths-and-payload/)
+
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+
+options:
+    state:
+        description:
+            - State of the custom policy.
+            - On V(present), the custom policy will be created (or updated if it exists already).
+            - On V(absent), the custom policy will be removed if it exists.
+        choices: ['present', 'absent']
+        default: 'present'
+        type: str
+    name:
+        description:
+            - Name of the custom policy to create.
+        type: str
+        required: True
+    policy_type:
+        description:
+            - The type of the policy. This must match the name of the custom policy deployed to the server.
+            - Multiple policies pointing to the same policy type can be created, but their names have to differ.
+        type: str
+        required: True
+    client_id:
+        description:
+            - The V(clientId) of the Keycloak client that should have the custom policy attached to it.
+            - This is usually a human-readable name of the Keycloak client.
+        type: str
+        required: True
+    realm:
+        description:
+            - The name of the Keycloak realm the Keycloak client is in.
+        type: str
+        required: True
+
+extends_documentation_fragment:
+    - community.general.keycloak
+    - community.general.attributes
+
+author:
+    - Samuli Seppänen (@mattock)
+'''
+
+EXAMPLES = '''
+- name: Manage Keycloak file:delete authorization scope
+  keycloak_authz_authorization_scope:
```

Comment:
The module's name is `keycloak_authz_custom_policy`, not `keycloak_authz_authorization_scope`. Also, please use FQCNs: `community.general.keycloak_authz_authorization_scope`.

Reply:
Done.